README.md

```diff
@@ -1,14 +1,19 @@
 # Project Tye
 
-Tye is a tool that makes developing, testing, and deploying microservices and distributed applications easier. Project Tye includes a local orchestrator to make developing microservices easier and the ability to deploy microservices to Kubernetes with minimal configuration. 
+Tye is a developer tool that makes developing, testing, and deploying microservices and distributed applications easier. Project Tye includes a local orchestrator to make developing microservices easier and the ability to deploy microservices to Kubernetes with minimal configuration. 
 
-What tye can do:
+Tye aims to:
 
-- Run multiple projects locally.
-- Easily manage dependencies running containers.
-- Automatically build Docker images by following .NET's conventions.
-- Deploy applications to Kubernetes with minimal configuration.
-- Make secret management and service discovery easy.
+- Simplify microservices development by making it easy to:
+  - Run whole applications
+  - Use dependencies in containers
+  - Discover addresses of other services using simple conventions
+- Deploy .NET applications to Kubernetes by:
+  - Automatially containerizing .NET applications
+  - Generating Kubernetes manifests with minimal knowledge or configuration
+  - Using the same conventions as development to keep it consistent
+
```

Comment:
Reorganized this to make it much more concrete. We got some *fair* criticism last release that this wasn't clear.